tldw_Server_API/app/api/v1/endpoints/media/process_pdfs.py

```
@@ -330,6 +370,57 @@ async def _pdf_batch_processor(process_items: List[ProcessItem]) -> List[Dict[st
+                        if isinstance(
+                            chunk_options_dict.get("hierarchical_template"), dict
+                        )
+                        else None,
+                    )
+                else:
+                    chunks = _improved_chunking_process(text, chunk_options_dict)
+
+                res["chunks"] = chunks
+    except Exception:
+        # Never fail the endpoint due to re-chunking issues.
+        pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)